apps/web/components/Proposals.tsx

```diff
@@ -377,118 +402,132 @@ export function Proposals({
+        <div>
+          <header className="flex items-center justify-between gap-10">
+            <h2>Proposals</h2>
+            {!proposals ? (
+              <LoadingSpinner />
+            ) : proposals.length === 0 ? (
+              <h4 className="text-2xl">No submitted proposals to support</h4>
+            ) : (
+              !allocationView && (
+                <Button
+                  icon={<AdjustmentsHorizontalIcon height={24} width={24} />}
+                  onClick={() => setAllocationView((prev) => !prev)}
+                  disabled={disableManSupportButton}
+                  tooltip={String(tooltipMessage)}
+                >
+                  Manage support
+                </Button>
+              )
             )}
           </header>
+          {allocationView && (
+            <>
+              <UserAllocationStats stats={stats} />
+            </>
+          )}
         </div>
 
         <div className="flex flex-col gap-6">
-          <div className="flex flex-col gap-6">
-            {proposals?.map((proposalData, i) => (
-              <React.Fragment key={proposalData.id + "_" + i}>
-                <ProposalCard
-                  proposalData={proposalData}
-                  inputData={inputs[i]}
-                  stakedFilter={stakedFilters[i]}
-                  i={i}
-                  isEditView={editView}
-                  tooltipMessage={tooltipMessage}
-                  memberActivatedPoints={memberActivatedPoints}
-                  memberPoolWeight={memberPoolWeight}
-                  executeDisabled={
-                    proposalData.proposalStatus == 4 ||
-                    !isConnected ||
-                    missmatchUrl
-                  }
-                  strategy={strategy}
-                  tokenDecimals={tokenDecimals}
-                  alloInfo={alloInfo}
-                  triggerRenderProposals={triggerRenderProposals}
-                  inputHandler={inputHandler}
-                />
-              </React.Fragment>
-            ))}
-          </div>
-          <div className="flex justify-end gap-8">
-            {editView && (
-              <>
-                <Button
-                  btnStyle="outline"
-                  color="danger"
-                  onClick={() => setEditView((prev) => !prev)}
-                >
-                  Cancel
-                </Button>
-                <Button
-                  onClick={() => submit()}
-                  isLoading={allocateStatus === "loading"}
-                  disabled={
-                    !getProposalsInputsDifferences(inputs, stakedFilters).length
-                  }
-                  tooltip="Make changes in proposals support first"
-                >
-                  Save changes
-                </Button>
-              </>
-            )}
-          </div>
+          {proposals?.map((proposalData, i) => (
+            <React.Fragment key={proposalData.id + "_" + i}>
```

Comment:
dont use index for key.
Reason: 
![image](https://github.com/user-attachments/assets/553cd4ef-f4a4-4822-a24e-0ad3231310a0)
https://legacy.reactjs.org/docs/lists-and-keys.html#keys

Reply:
Added a rule with warn level in my PR